pymc3/tests/test_distributions.py

```diff
@@ -1371,28 +1398,38 @@ def test_binomial(self):
     @pytest.mark.xfail(
         condition=(SCIPY_VERSION < parse("1.4.0")), reason="betabinom is new in Scipy 1.4.0"
     )
-    def test_beta_binomial(self):
-        self.checkd(
+    def test_beta_binomial_logcdf(self):
+        self.check_logcdf(
             BetaBinomial,
             Nat,
             {"alpha": Rplus, "beta": Rplus, "n": NatSmall},
+            lambda value, alpha, beta, n: sp.betabinom.logcdf(value, a=alpha, b=beta, n=n),
+            n_samples=-1,
```

Comment:
Xfailed as expected in https://github.com/pymc-devs/pymc3/pull/4497/checks?check_run_id=2021551266